.github/workflows/build-push-image.yaml

```diff
@@ -0,0 +1,194 @@
+name: Build Image and Push to Registries
```

Comment:
can we update the existing workflow and make it more configurable instead of having 2 different workflows for the same thing? https://github.com/uc-cdis/.github/blob/master/.github/workflows/image_build_push.yaml